src/installer.ts

```diff
@@ -145,10 +173,10 @@ export async function installKamel(context: vscode.ExtensionContext): Promise<Er
 
 	let versionToUse = latestversion.result.trim();
 
-	await checkKamelCLIVersion().then((currentVersion) => {
-		if (versionToUse.toLowerCase() === currentVersion.toLowerCase()) {
+	await checkKamelNeedsUpdate(versionToUse).then((needsUpdate) => {
+		if (needsUpdate) {
 			// no need to install, it's already here
```

Comment:
this comment is outdated. In this case, the version needs to be downloaded and updated